proton-c/src/tests/fuzz/fuzz-sniff-header.c

```diff
@@ -0,0 +1,15 @@
+#include <stdlib.h>
+#include <stdio.h>
+
+#include "core/autodetect.h"
+
+#ifdef __cplusplus
```

Comment:
Even so, you shouldn't need this outside header files. whether the internal symbol names are C++ or C is not important unless they are visible externally.